### ISSUE TITLE
Add rarity table for word selection

### ISSUE THREAD

**Issue by ryanclanigan** [ryanclanigan/messaging-bridge#8]

**Comment by ryanclanigan** [comment]
Done in 50f1e2f75e55360a41e81bfa2f07cde81f3ee659

**Comment by Aqua-duck** [comment]
Would it be possible for us to make it so that users can run a command to add a new verb + weight using the discord bot (or both, if its possible on messenger)?
E.g. I type something like `!adverb pooped 0.3` to add "pooped" to the table with a probability of 30%

**Comment by ryanclanigan** [comment]
Oooh that's an interesting idea. The only problem is that by saying you
give something a 30% of happening, what does that mean for the other words
that already have rarities. Ideally it would also have a way to remove
words as well. That would tie in with my idea of making the starting list
be an editable file.

Perhaps you can give each phrase an arbitrary value? And that determines
how likely it actually is.

**Comment by Aqua-duck** [comment]
Yeah the method of assigning weight to the values can be different, I just used percentages as an example. Preface: I don't know if this is bad practice or would lead to performance issues. But maybe we could just assign each phrase an integer and that translates to how frequently it would "appear" in the list when it's searched by the random function.

E.g. `!adverb poop 5` makes it so that "pooped" would effectively be 5x more likely than a phrase with a value of 1

And yes, a way to remove phrases would also be very useful. As a supplement to that we could have a command that prints the current list of phrases along with their weights
